test/builders/build/eth-api/libraries/viem/send-transaction.js

```diff
@@ -0,0 +1,66 @@
+import { assert } from 'chai';
+import { createPublicClient, createWalletClient, http, parseEther } from 'viem';
+import { privateKeyToAccount } from 'viem/accounts';
+import { moonbaseAlpha } from 'viem/chains';
+
+const fromAddressPrivateKey = "0x5fb92d6e98884f76de468fa3f6278f8807c48bebc13595d45af5bdc4da702133"
+const fromAddress = "0xf24FF3a9CF04c71Dbc94D0b566f7A27B94566cac"
```

Comment:
I don't think this is necessary; there seem to be extra variables.

You essentially are doing

```
const fromAddress = "0xf24FF3a9CF04c71Dbc94D0b566f7A27B94566cac"
const addressFrom = fromAddress
```

no need for the extra variable, you could just do:

```
const addressFrom = '0xf24FF3a9CF04c71Dbc94D0b566f7A27B94566cac';
```

Reply:
same with `toAddress`